ci/common.sh

```diff
@@ -25,7 +25,8 @@ export_or_prefix() {
     export OPENRESTY_PREFIX="/usr/local/openresty"
     export APISIX_MAIN="https://raw.githubusercontent.com/apache/apisix/master/apisix-master-0.rockspec"
     export PATH=$OPENRESTY_PREFIX/nginx/sbin:$OPENRESTY_PREFIX/luajit/bin:$OPENRESTY_PREFIX/bin:$PATH
-    export OPENSSL111_BIN=$OPENRESTY_PREFIX/openssl111/bin/openssl
+    export OPENSSL_PREFIX=$OPENRESTY_PREFIX/openssl3
+    export OPENSSL_BIN=$OPENSSL_PREFIX/bin/openssl
```

Comment:
Not openssl3?

Reply:
Openssl3 is the directory, and openssl is a binary program.